en/docs/learn/deploying-the-sample-app.md

```diff
@@ -629,7 +629,12 @@ To check the **photo-edit** sample application, navigate to
 on your browser.
 
 For example,
-`                     http://localhost.com:8080/photo-edit/index.jsp                 .         `
+`                     http://localhost.com:8080/photo-edit/index.jsp                 . 
+
+!!! note
+    -   The default password configuration for photo-editor and photo-viewer is `admin`. If a different password needs to be used to test the samples, change the hardcoded password in `<TOMCAT_HOME>/webapps/photo-view/WEB-INF/classes/apptwo.properties` or `<TOMCAT_HOME>/webapps/photo-edit/WEB-INF/classes/apptwo.properties`. 
+
+    -    All the directories of the tomcat server should be accessible even without root access. Otherwise, the application fails due to insufficient permissions. `
```

Comment:
Shall we give a tip here suggesting to start the Tomcat server using `sudo` (for Mac) or `run as administrator` in Windows to avoid root access permission issues?

Reply:
@sherenewso2 Starting the tomcat with 'sudo' (root) didn't solve the permission issue for me, after setting up the correct directory permissions it started functioning well. 
Please refer to https://superuser.com/questions/632618/best-practice-for-access-permission-to-users-for-apache-tomcat

Reply:
I am trying to access both sample apps with 'admin' credentials and with the suggested ones from here https://is.docs.wso2.com/en/5.11.0/learn/configuring-access-delegation-with-uma/. Sample apps both runs fine but in photo-view app there is no album after sharing. Any suggests?